pkg/services/rpcsrv/server.go

```diff
@@ -153,6 +153,7 @@ type (
 		notificationCh  chan *state.ContainedNotificationEvent
 		transactionCh   chan *transaction.Transaction
 		notaryRequestCh chan mempoolevent.Event
+		subEventCh      chan struct{}
```

Comment:
How about s/`subEventCh`/`subEventsToExitCh`? It's kind of convention within the whole set of neo-go exit channels.

Reply:
renamed